### PR TITLE
Added provider for Evernote

### PR DESCRIPTION
**Pull request by CarteKiwi**
Evernote provider has been added.
Tested in a Xamarin project for iOS and Android.

### CODE REVIEW

File: src/Owin.Security.Providers.Evernote/EvernoteAuthenticationOptions.cs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        public ISecureDataFormat<RequestToken> StateDataFormat { get; set; }
+
+        /// <summary>
+        ///     Initializes a new <see cref="EvernoteAuthenticationOptions" />
+        /// </summary>
+        public EvernoteAuthenticationOptions()
+            : base("Evernote")
+        {
+            Caption = Constants.DefaultAuthenticationType;
+            CallbackPath = new PathString("/signin-Evernote");
```
Comment (TerribleDev) [inline src/Owin.Security.Providers.Evernote/EvernoteAuthenticationOptions.cs]: Can you make the entire default url lowercase?

**Comment by TerribleDev** [comment]
@CarteKiwi thank you for the contribution! When I have time (in the next day or two ish) I'll look this over and give some feedback.

**Comment by CarteKiwi** [comment]
You're welcome. Thanks YOU for this repository :)

File: src/Owin.Security.Providers.Evernote/EvernoteAuthenticationOptions.cs (hunk 1 above)
Comment (CarteKiwi) [reply src/Owin.Security.Providers.Evernote/EvernoteAuthenticationOptions.cs]: Done

**Comment by TerribleDev** [comment]
holy crap I TOTALLY forgot about this pr sorry!

**Comment by TerribleDev** [comment]
@CarteKiwi sorry I didn't merge sooner. Now I feel bad 😞

**Comment by CarteKiwi** [comment]
@TerribleDev it's ok, don't worry. Better late than never ;)

**Pull request merged**